src/lib/notificationBroadcasterWebsocket.js

```diff
@@ -0,0 +1,59 @@
+'use strict'
+
+const _ = require('lodash')
+const EventEmitter = require('events').EventEmitter
+const transferDictionary = require('five-bells-shared').TransferStateDictionary
+const transferStates = transferDictionary.transferStates
+const isTransferFinalized = require('./transferUtils').isTransferFinalized
+const convertToExternalTransfer = require('../models/converters/transfers')
+  .convertToExternalTransfer
+const getFulfillment = require('../models/db/fulfillments').getFulfillment
+const convertToExternalFulfillment = require('../models/converters/fulfillments')
+  .convertToExternalFulfillment
+
+class NotificationBroadcaster extends EventEmitter {
+  constructor (log, config) {
+    super()
+    this.log = log
+    this.config = config
+  }
+
+  * sendNotifications (transfer, transaction) {
+    const affectedAccounts = _([transfer.debits, transfer.credits])
+      .flatten().pluck('account').value()
+    affectedAccounts.push('*')
+
+    // Prepare notification for websocket subscribers
+    const notificationBody = {
+      resource: convertToExternalTransfer(transfer)
+    }
+
+    // If the transfer is finalized, see if it was finalized by a fulfillment
+    let fulfillment
```

Comment:
For Universal it is essential that the fulfillment be included with the notification. The connector needs to get that as quickly as possible to pass it on and doing a `GET` would make it much slower